### PR TITLE
Adding Candied Carrots Mar Recipe

### PR DESCRIPTION
**Pull request by brianbudowick**
Merging Changed for the Carrots

### CODE REVIEW

File: app/index.md
Comment (emileastih1) [inline app/index.md:1]: Brian here it seems that you removed the Mac Gump Writer recipe.
You shouldn't do that, instead you should just append your changes in the end of the file.

**Comment by emileastih1** [comment]
Please correct the wrong file

File: app/index.md
Comment (emileastih1) [reply app/index.md:1]: ok Thank you brian for your fixed it is fine

Comment (leedsa123) [inline app/index.md:1]: Brian, It appears that your recipe is flawless

**Review by leedsa123 — COMMENTED** [review]
très bien

**Pull request merged**